app/components/gleam-logo.ts

```
@@ -0,0 +1,106 @@
+          fit: Fit.Contain,
+        }),
+        onLoad: () => {
+          if (this.riveInstance) {
+            // Initial resize
+            this.riveInstance.resizeDrawingSurfaceToCanvas();
+
+            const inputs = this.riveInstance.stateMachineInputs('State Machine 1');
+
+            // Try to trigger hover state
+            if (inputs) {
+              inputs.forEach(input => {
```

ESLint

[error] Replace `input` with `(input)`

(prettier/prettier)